man/systemd.mount.xml

```diff
@@ -359,6 +359,17 @@
         <varname>Options=</varname> setting in a unit file.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>x-systemd.no-remount-ro</option></term>
+
+        <listitem><para>If a mount operation fails to mount the file system
+        read-write, it normally tries to remount the file system read-only.
```

Comment:
I think "remount" is actually a less than ideal choice of words. We usually use "remount" for the operation when you have a mount already and you change its parameters. But in this case this is not what happens, the mount is not there, and we try to establish it once unsuccessfully, so we *retry* it in a slightly modified way. Hence should say "retries to mount"

Reply:
Rephrased a bit to use "retry" instead of "remount"